src/JustSaying/Extensions/ChunkExtensions.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System.Diagnostics;
+
+namespace JustSaying.Extensions;
```

Comment:
You could put this into the same namespace as the original code, then with a hash define we can just not include it at all and use the built-in version.